pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.marklogic</groupId>
   <artifactId>marklogic-mule-connector</artifactId>
-  <version>2.0.0-SNAPSHOT</version>
+  <version>2.0.113-SNAPSHOT</version>
```

Comment:
Looks like a typing error.

Reply:
Ah, that's me forgetting to roll that back to 2.0.0 - that gives you an idea of how many times I've been publishing locally lately. :)